README.md

```diff
@@ -170,3 +170,10 @@ nebula-operator refers to [tidb-operator](https://github.com/pingcap/tidb-operat
 ## License
 
 NebulaGraph is under the Apache 2.0 license. See the [LICENSE](./LICENSE) file for details.
+
+
+
+| Nebula Operator版本 | Nebula Graph版本 |
+| ------------------- | ---------------- |
+| v0.8.0              | v2.5.0           |
+
```

Comment:
Can you use english?

And need to adjust the position instead of putting it in the License?